### PR TITLE
Add some prose to the blockchain layer secion of the Shelley spec

### PR DESCRIPTION
**Pull request by mgudemann**

### CODE REVIEW

File: latex/chain.tex [hunk 1]
```diff
@@ -218,6 +233,30 @@ \subsection{Block Definitions}
...
+\item The number of the last epoch.
+\item The old epoch nonce.
+\item The information about produced blocks for each stake pool.
+\item The old epoch state.
+\item An optional rewards update.
+\item The stake pool distribution of the epoch.
+\end{itemize}
+
+\begin{note}
+  Where is \PoolDistr defined?
```
Comment (JaredCorduan) [inline latex/chain.tex]: I think we can remove this note since you added a definition above on line 100.

File: latex/chain.tex [hunk 2]
```diff
@@ -694,10 +931,13 @@ \subsection{Block Body Transition}
       \var{txs} \leteq \bbody{block}
       &
       \var{vk} \leteq \bissuer{bhb}
+      &
+      \var{hk} \leteq \hashKey{vk}
       \\
-      \bSize{bh} < \fun{maxBBSize}~\var{pp}
```
Comment (JaredCorduan) [inline latex/chain.tex]: yes,` bBodySize` is correct. I think this method is called `bSize` in the byron spec.

File: latex/chain.tex [hunk 3]
```diff
@@ -694,10 +931,13 @@ \subsection{Block Body Transition}
...
+      \bBodySize{txs} < \fun{maxBBSize}~\var{pp} % TODO is bBodySize the right
+                                % thing here?
       &
-      \var{vk}\mapsto n\in (b \unionoverrideLeft \{\var{vk}\mapsto 0\})
+      \var{hk}\mapsto n\in (b \unionoverrideLeft \{\var{vk}\mapsto 0\})
       \\~\\
```
Comment (JaredCorduan) [inline latex/chain.tex]: It looks like we are still missing a signature check here. we can get `\sigma := bsig bhb`, and then verify the `\sigma` with `vk` against `txs`.  we can do this now if it's convenient, or I'll do it next.

Comment (mgudemann) [reply latex/chain.tex]: added the signature check for `txs`

**Pull request merged**